react-hot-button-component/src/hot-button.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+class HotButton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { clicks: 0 };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
+    this.setState({ clicks: this.state.clicks + 1 });
+  }
+
+  render() {
+    if (this.state.clicks >= 3 && this.state.clicks < 6) {
```

Comment:
While this works fine, I recommend creating a method which will return the proper classname based on clicks when called. Then call that method in render, save the classname which is returned, and then use it to set the classname for the button. This will clean up the render method, and save you from having to return many versions of essentially the same thing.